src/model/fsm_util.jl

```diff
@@ -184,22 +184,65 @@ end
 
 
 """
-Populate fs0m_vec and fs1m_vec for all sources for a given pixel.
+Populate fs0m and fs1m for source s in the a given pixel.
 
-Args:
-    - model_vars: Model intermediate values.
```

Comment:
Maybe it would be easier to fit these renaming requests into less substantive PRs.